src/NodeRTLib/ProjectFiles/NodeRtUtils.cpp

```diff
@@ -316,9 +316,7 @@ namespace NodeRT { namespace Utils {
     }
 
     v8::String::Value stringVal(value);
-    std::wstring guidStr( L"{" );
-    guidStr += StringToWchar(stringVal);
-    guidStr += L"}" ;
+    std::wstring guidStr = StringToWchar(stringVal);
```

Comment:
Sorry about the late comment, but Ideally NodeRT should support both cased - with and without the brackets. The fix should take this into account and test whether the string starts/end with a bracket.